tools/create-yorkie-app/index.ts

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env node
+
+/* eslint-disable jsdoc/require-jsdoc */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import minimist from 'minimist';
+import prompts from 'prompts';
+import { red, reset } from 'kolorist';
+import { type Framework, FRAMEWORKS } from './frameworks';
+
+// Avoids autoconversion to number of the project name by defining that the args
+// non associated with an option ( _ ) needs to be parsed as a string. See https://github.com/vitejs/vite/pull/4606
+const argv = minimist<{
+  t?: string;
+  template?: string;
+}>(process.argv.slice(2), { string: ['_'] });
+const cwd = process.cwd();
+
+const renameFiles: Record<string, string | undefined> = {
+  _gitignore: '.gitignore',
+};
+
+const TEMPLATES = FRAMEWORKS.map(
+  (f) => (f.variants && f.variants.map((v) => v.name)) || [f.name],
+).reduce((a, b) => a.concat(b), []);
+
+const apiKeyMessage = 'You can update your API key in .env';
+let apiKey = '';
+
+const defaultTargetDir = 'yorkie-app';
+
+async function init() {
+  const argTargetDir = formatTargetDir(argv._[0]);
+  const argTemplate = argv.template || argv.t;
+
+  let targetDir = argTargetDir || defaultTargetDir;
+  const getProjectName = () =>
+    targetDir === '.' ? path.basename(path.resolve()) : targetDir;
+
+  let result: prompts.Answers<
+    | 'apiKey'
+    | 'projectName'
+    | 'overwrite'
+    | 'packageName'
+    | 'framework'
+    | 'variant'
+  >;
+
+  try {
+    result = await prompts(
+      [
+        {
+          type: () => {
+            console.log(
+              '\n🔑 To run these examples, you need Yorkie API key.' +
+                '\nGet your API key at https://yorkie.dev/dashboard\n',
+            );
+
+            return 'text';
+          },
+          name: 'apiKey',
+          message: reset('API key:'),
+          initial: apiKeyMessage,
+          onState: ({ value }) => {
+            apiKey = value === apiKeyMessage ? '' : value;
+          },
```

Comment:
I think this step is not applicable to all examples. for example, next.js scheduler example doesn't use vite, so we need to declare an environment variable like `NEXT_PUBLIC_YORKIE_API_ADDR`, but this s step will overwrite the example's `.env` with vite-specific environment variable.

Suggestion: How about remove this step and print that message about the api key after the project is created?